src/main/java/randoop/types/NonParameterizedType.java

```diff
@@ -175,7 +175,9 @@ public boolean isBoxedPrimitive() {
 
   @Override
   public boolean isEnum() {
-    return runtimeType.isEnum();
+    return runtimeType.isEnum()
+        || (getRuntimeClass().getSuperclass() != null
+            && getRuntimeClass().getSuperclass().isEnum());
```

Comment:
I was surprised by this code, since users cannot subclass an enum class.  However, `runtimeType` might be an enum _constant_ (as opposed to a class declared as `enum`).  I think this is worth a brief comment given my initial surprise.

For more details, https://docs.oracle.com/javase/specs/jls/se22/html/jls-8.html#jls-8.9, says "The permitted direct subclasses ([§8.1.6](https://docs.oracle.com/javase/specs/jls/se22/html/jls-8.html#jls-8.1.6)) of E are the anonymous classes implicitly declared by the enum constants that have a class body."